src/nccl_ofi_rdma.c

```diff
@@ -5431,8 +5506,12 @@ static int release_ep(nccl_net_ofi_ep_t *base_ep)
 	 * deallocation.
 	 */
 	if (ep->ref_cnt == 0) {
-		/* Ideally we would "un-post" the bounce buffers, but this
-		   should be accomplished by closing the endpoint. */
+
+		ret = cancel_posted_recv_ops(ep);
```

Comment:
you should have a comment here about why we're doing this.

Reply:
Also, please confirm you've opened the two tickets we talked about with the Libfabric team?

Reply:
> you should have a comment here about why we're doing this.
Done

> Also, please confirm you've opened the two tickets we talked about with the Libfabric team?
Confirmed.